packages/react-select/src/Select.tsx

```diff
@@ -1738,7 +1740,9 @@ export default class Select<
             'aria-describedby': this.getElementId('live-region'),
           }
         : {
-            'aria-describedby': this.getElementId('placeholder'),
+            'aria-describedby':
+              this.props['aria-describedby'] ||
+              this.getElementId('placeholder'),
```

Comment:
Perhaps this should be:

```suggestion
            'aria-describedby': [
              this.props['aria-describedby'],
              this.getElementId('placeholder'),
            ]
              .filter(Boolean)
              .join(' '),
```